server side/router/visitedWebsite.router.js

```diff
@@ -0,0 +1,9 @@
+import express from 'express'
+import {getAllVisitedWebsites,getVisitedWebsiteById,deleteVisitedWebsite,updateVisitedWebsite,createVisitedWebsite} from '../controllers/visitedWebSiteController'
+const router=express.Router();
+router.get('/getAllVisitedWebsites',getAllVisitedWebsites)
+router.get('/getVisitedWebsiteById/:id',getVisitedWebsiteById)
+router.post('/createVisitedWebsite',createVisitedWebsite)
+router.put('/updateVisitedWebsite/:id',updateVisitedWebsite)
+router.delete('/deleteVisitedWebsite/:id',deleteVisitedWebsite)
```

Comment:
remove the create-update-delete words, look in the user router for the correct way to do it.